carma/launch/environment.launch.py

```diff
@@ -33,6 +33,7 @@ def generate_launch_description():
     """
     Launch perception nodes.
     """
+    vehicle_calibration_dir = LaunchConfiguration('vehicle_calibration_dir')
```

Comment:
I thought we had to use DeclareLaunchArgument as well in order to have "opt/carma/vehicle/calibration" passed from carma_src.launch.py, but I might be wrong. Does this work?

Reply:
That makes sense! Very good to know, and I hadn't realized we used this same approach in guidance.launch.py already.